actions/class.RestQtiTests.php

```diff
@@ -212,4 +237,12 @@ private function getTestClass()
     {
         return $this->getClassFromRequest(new \core_kernel_classes_Class(TaoOntology::CLASS_URI_TEST));
     }
+
+    /**
+     * @return QtiPackageExporter
+     */
+    private function getQtiPackageExporter()
```

Comment:
Good work! One thing I'd suggest is maybe it would be better to use return type hints instead of @return here and in other places, if it allowed by target PHP version (did not found version declaration in composer.json). What do you think?

Reply:
I wish I could. Unfortunately because of some issues, I can't use type hinting. AFAIK